MedicalBfr.php

```diff
@@ -65,8 +65,7 @@ function printit(){window.print();}
 <?php
 $DEBUG=0;
 $diagtext="";
-$con_params = require('./config/database.php'); $con_params = $con_params['gliding']; 
-$con=mysqli_connect($con_params['hostname'],$con_params['username'],$con_params['password'],$con_params['dbname']);
+$con=mysqli_connect("127.0.0.1","admin","Checkers305","gliding");
```

Comment:
@tim-hogan what is wrong with using ```$con_params```? is it failing to connect under certain circumstances?
I really do not think it is a good idea to have the  database admin password published in GIT.